_data/cryptocurrencies.yml

```
@@ -147,108 +134,107 @@ websites:
     url: https://btcpop.co/
     img: btcpop.png
     tfa:
-      - software
+      - totp
     doc: https://blog.btcpop.co/2017/04/27/signing-security-btcpop-co-account/
 
   - name: BX Thailand
     url: https://bx.in.th/
     img: bxthailand.png
     tfa:
-      - software
+      - totp
     doc: https://bx.in.th/info/faq/
 
   - name: CEX.IO
     url: https://cex.io/
     img: cex.png
     tfa:
       - sms
-      - software
+      - totp
     doc: https://cex.io/guide#4
 
   - name: Changelly
     url: https://changelly.com/
     img: changelly.png
     tfa:
-      - software
+      - totp
     doc: https://changelly.com/faq#what-is-2fa-authentication
 
   - name: Circle
     url: https://circle.com
     img: circle.png
     tfa:
       - sms
-      - software
+      - totp
     doc: https://support.circle.com/hc/en-us/articles/213560483
 
   - name: COBINHOOD
     url: https://cobinhood.com
     img: cobinhood.png
     tfa:
       - sms
-      - software
+      - totp
     doc: https://cobinhood.zendesk.com/hc/en-us/articles/115003476871
 
   - name: Coin Cafe
     url: https://coincafe.com
     img: coincafe.png
     tfa:
       - email
-      - software
+      - totp
     doc: https://coincafeinc.wordpress.com/2016/01/08/2fa/
 
   - name: Coinapult
     url: https://coinapult.com
     img: coinapult.png
     tfa:
-      - software
+      - totp
     doc: https://coinapult.com/faq
 
   - name: Coinbase
     url: https://www.coinbase.com/
     img: coinbase.png
     tfa:
       - sms
-      - phone
-      - software
-      - hardware
+      - totp
+      - u2f
     doc: https://support.coinbase.com/customer/en/portal/articles/1658338
 
   - name: CoinDeal
     url: https://coindeal.com
     img: coindeal.png
     tfa:
       - sms
-      - software
+      - totp
     doc: https://coindeal.com/security
 
   - name: CoinFalcon
     url: https://coinfalcon.com/
     img: coinfalcon.png
     tfa:
-      - software
+      - totp
     doc: https://support.coinfalcon.com/en-us/article/how-to-setup-two-factor-authentication-2fa-nkjppd/
```

```suggestion
    doc: https://support.coinfalcon.com/en-us/article/-nkjppd/
```
Shorter URL possible.